src/Tribe/Tickets.php

```diff
@@ -3225,14 +3234,12 @@ public function maybe_redirect_to_attendees_registration_screen( $redirect = nul
 
 			$url = $attendee_reg->get_url();
 
-			$provider = tribe_get_request_var( 'provider' );
-
-			if ( empty( $provider ) ) {
-				$provider = $this->attendee_object;
+			if ( empty( $q_provider ) ) {
```

Comment:
This isn't needed anymore after adding the conditional to set it above

Reply:
good catch